packages/d3fc-series/src/webgl/bar.js

```diff
@@ -30,7 +30,7 @@ export default () => {
             .widths(widths)
             .xScale(xScale.glScale)
             .yScale(yScale.glScale)
-            .decorate((program) => {
+            .decorate(program => {
```

Comment:
Same as above, we have brackets here in the other implementations

Reply:
Done.